seeds/create_dashboard.js

```diff
@@ -0,0 +1,25 @@
+// if our user.js file is at models/user.js
```

Comment:
Please remove seed files; we should be adding data through a UI, or API endpoints (which is a good next step here)